### DISCUSSION TITLE
Direct repo access for contributors

### DISCUSSION THREAD

**Discussion by rmarquis** [etlegacy/etlegacy#1721] [General]
I took the liberty to give both @ryzyk-krzysiek and @Aciz direct write access to the master branch, since they've been quite active lately. Hope nobody minds.

You can still push PR if some changes require discussion and review, or open a new ticket for discussion if needed, but feel free to push directly in master if the changes are no-brainer.

**Comment by Aranud** [comment]
Was in my mind to do so, I'm glad you did it !

**Comment by ryzyk-krzysiek** [comment]
> Hope nobody doesn't mind.

I do :p I'm terrible at git, basically learing it with this project so I hope what I was doing up to this point is not affected and at worse case I will only break my fork.

**Reply by Aciz** [reply]
> I do :p I'm terrible at git, basically learing it with this project so I hope what I was doing up to this point is not affected and at worse case I will only break my fork.

Just keep doing stuff via PRs and you have nothing to fear :)

**Reply by Aranud** [reply]
No obligation indeed ! You just gain a total liberty, no constraint, no obligation.

**Reply by rmarquis** [reply]
Just keep doing what you were doing. You basically can now merge your own PR, so we don't have to do that chore by ourselves if our input isn't required.

I'd still suggest to read a book or some tutorials about git, it's a fantastic tool and learning more about it will serve you well in the future.
